backend/tests/unit/utils/test_template_resolver_env_file.py

```
@@ -0,0 +1,442 @@
+                # Should still work with other sources
+                with patch.dict(os.environ, {'TEST_VAR': 'system_value'}):
+                    config = {"key": "${TEST_VAR}"}
+                    result = resolver.resolve_configuration(config)
+                    assert result == {"key": "system_value"}
+                    
+            finally:
+                # Restore permissions and clean up
+                try:
+                    os.chmod(f.name, 0o644)
+                    os.unlink(f.name)
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)